### PR TITLE
Buildkite: Run jormungandr integration tests in the nightly pipeline

### PR DESCRIPTION
**Pull request by rvl**
Runs a stack build with jormungandr integration tests unskipped as part of the nightly build.

[Buildkite pipeline for this branch Build status](https://buildkite.com/input-output-hk/cardano-wallet-nightly?branch=rvl%2Fbuildkite-integration-tests-nightly)

### CODE REVIEW

**Review by KtorZ — APPROVED** [review]
Cool. We don't have to create a JIRA ticket for that then :sweat_smile:

**Comment by rvl** [comment]
Perhaps we can add a CI step to create a JIRA ticket if it doesn't already exist.

I am not sure yet what to do if the jormungandr tests fail. Currently it makes the nightly red if this happens.

**Comment by rvl** [comment]
@Anviking I moved the "queue: benchmark" setting to the benchmark steps in the yaml file.
If you set a queue on the upload pipeline step, only that step runs on the chosen queue.

File: .buildkite/nightly.yml [hunk 1]
```diff
@@ -44,3 +48,12 @@ steps:
     timeout_in_minutes: 60
     agents:
       system: x86_64-linux
+
+  - label: "Rebuild master branch"
+    trigger: "cardano-wallet"
+    branches: "master"
+    build:
+      message: "Nightly build with jormungandr integration tests"
```
Comment (Anviking) [inline .buildkite/nightly.yml:55]: There's no `commands` here, is it triggering the normal `.buildkite/rebuild` somehow?

Comment (rvl) [reply .buildkite/nightly.yml:55]: Yes, it's meant to trigger the other pipeline. I just got it working. You can see what happens on Buildkite.

Comment (rvl) [reply .buildkite/nightly.yml:55]: This is what it does:
Screenshot from 2020-04-23 12-11-15

**Comment by piotr-iohk** [comment]
@rvl should there be some results on https://buildkite.com/input-output-hk/cardano-wallet-nightly from last night, I cannot find anything... :sweat_smile:

**Comment by rvl** [comment]
@piotr-iohk Yeah you have to click through the link "Rebuild master branch" down the bottom.

Here is an example:

https://buildkite.com/input-output-hk/cardano-wallet/builds/7128#307c9329-5250-4304-9ebe-99de857a9266/728-794

**Pull request merged**